src/contracts/core/DelegationManager.sol

```diff
@@ -651,16 +666,16 @@ contract DelegationManager is
             IShareManager shareManager = _getShareManager(strategies[i]);
 
             // delegatedShares for staker to place into queueWithdrawal
-            delegatedSharesToWithdraw[i] = WithdrawableShares.wrap(sharesToWithdraw[i]).toDelegatedShares(totalMagnitudes[i]);
-            uint256 principalSharesToWithdraw = Shares.unwrap(
-                delegatedSharesToWithdraw[i].toShares(depositScalingFactors[staker][strategies[i]])
-            );
+
+            delegatedSharesToWithdraw[i] = sharesToWithdraw[i].toDelegatedShares(totalMagnitudes[i]);
+            
+            Shares principalSharesToWithdraw = delegatedSharesToWithdraw[i].toShares(depositScalingFactors[staker][strategies[i]]);
```

Comment:
principalShares

Reply:
what mean here?

Reply:
delegatedSharesToWithdraw -> sharesToWithdraw?

Reply:
principalSharesToWithdraw -> sharesToWithdraw

there should be no mention of principal